pkg/services/reputation/local/storage/calls.go

```diff
@@ -40,14 +41,17 @@ type trustValue struct {
 // EpochTrustValueStorage represents storage of
 // the trust values by particular epoch.
 type EpochTrustValueStorage struct {
+	reputationcommon.ServerInfo
```

Comment:
Why do we need this?

Reply:
after the addition `trusting` peer to `Trust` struct there was bug related to `Trust`s without(zero) `trusting` peer in storages

in fact, i think that it is not so necessary in that case, but to prevent potential bugs in the future, i suggest creating all local `Trust` with trusting peer(via ServerInfo in that case) always

it is used at 165 line

Reply:
deleted